### ISSUE TITLE
Can ranger.desktop open every file in the same Ranger instance?

### ISSUE THREAD

**Issue by mrbig033** [ranger/ranger#1239]
#### Runtime Environment

- Operating system and version: [Manjaro (stable)](https://manjaro.org)
- Terminal emulator and version: [xfce4-terminal 0.8.7.4 (Xfce 4.12)](https://docs.xfce.org/apps/terminal/start)
- Python version: Python 3.6.6 (default, Jun 27 2018, 13:11:40)
- Ranger version/commit: ranger-master 1.9.1
- [My Ranger files](https://gitlab.com/mrbig/dotfiles/tree/master/ranger)
- Locale: Locale: en_US.UTF-8
- [i3 version 4.15 (2018-03-10)](https://i3wm.org/)
- [i3/config](https://gitlab.com/mrbig/dotfiles/blob/master/i3/config-manjaro)
- ranger.desktop - default
```
[Desktop Entry]
Type=Application
Name=ranger
Comment=Launches the ranger file manager
Icon=utilities-terminal
Terminal=true
Exec=ranger
Categories=ConsoleOnly;System;FileTools;FileManager
MimeType=inode/directory;
```
- ranger.desktop - xfce4-terminal
```
[Desktop Entry]
Type=Application
Name=ranger
Comment=Launches the ranger file manager
Icon=utilities-terminal
Terminal=true
Exec= xfce4-terminal -T ranger-desktop -x ranger
Categories=ConsoleOnly;System;FileTools;FileManager
MimeType=inode/directory;
```
#### Current Behavior

I was able to make my system correctly open files in Ranger with both desktop entries, but it opens a new instance every time I open a file. This is not a bug, but I wish I had the option to call a single Ranger instance instead.

#### Expected Behavior

Having the option to allow only a single Ranger instance.

#### Context

I think preventing the multiplication of Ranger instances would be a better workflow for a tiling window manager like i3.

#### Possible Solutions

I think a simple script could solve this.

#### Steps to reproduce
1. Run the command `xdg-mime default ranger.desktop inode/directory`
2. Download any file on a web browser like Google Chrome
3. Click to "show in folder" or similar command.
4. Repeat. Ranger should open one instance for each command.

**Comment by toonn** [comment]
"4. ... for every command." you mean? Ranger supports some IPC so I think this is possible.

**Comment by mrbig033** [comment]
Yeah, currently another instance is opened with every command. My understanding of bash is very limited and I never used IPC commands for something like Ranger, but I can give it a try. Do you have any suggestions?

**Comment by Vifon** [comment]
You can try using this plugin for this: https://github.com/ranger/ranger/blob/master/examples/plugin_ipc.py
